### PR TITLE
chore: silence logic test logs

### PR DESCRIPTION
**Pull request by ZeaLoVe**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Silence logic test logs, change statements log level to debug, issue #6596 

Fixes #issue

### CODE REVIEW

File: tests/logictest/logictest.py [hunk 1]
```diff
@@ -281,28 +281,26 @@ def execute(self):
         if callable(getattr(self, "batch_execute")):
             # case batch
             for (file_path, suite_name) in self.statement_files:
-                log.info("Batch execute, suite name:{} in file {}".format(
-                    suite_name, file_path))
+                log.info("Run query with the same session every suite file, suite file path:{}".format(file_path))
```
Comment (Xuanwo) [inline tests/logictest/logictest.py:284]: As we are using python 3.10, how about using format strings for better reading, like:

```python
log.info(f"Run query with the same session every suite file, suite file path:{file_path}")
```

File: tests/logictest/logictest.py [hunk 2]
```diff
@@ -281,28 +281,26 @@ def execute(self):
...
                 statement_list = list()
                 for state in get_statements(file_path, suite_name):
                     statement_list.append(state)
                 self.batch_execute(statement_list)
         else:
             # case one by one
             for (file_path, suite_name) in self.statement_files:
-                log.info("One by one execute, suite name:{} in file {}".format(
-                    suite_name, file_path))
+                log.info("Run query without session every statements, suite file path:{}".format(file_path))
```
Comment (Xuanwo) [inline tests/logictest/logictest.py:292]: ditto

File: tests/logictest/logictest.py (hunk 1 above)
Comment (Xuanwo) [reply tests/logictest/logictest.py:284]: BTW, it's better to add a space after `:`, which will make it easier for reading.

File: tests/logictest/logictest.py (hunk 2 above)
Comment (ZeaLoVe) [reply tests/logictest/logictest.py]: > ditto

PTAL

**Review by Xuanwo — APPROVED** [review]

**Review by BohuTANG — APPROVED** [review]

**Pull request merged**